ui-tests/src/test/java/io/syndesis/qe/steps/customizations/api_client_connectors/ApiClientConnectorsSteps.java

```diff
@@ -0,0 +1,180 @@
+package io.syndesis.qe.steps.customizations.api_client_connectors;
```

Comment:
Pls rename the package to only e.g. `connectors`, no underscores or dashes.

Reply:
renamed